tests/load-tests/ci-scripts/utility_scripts/playwright-create-users.py

```diff
@@ -0,0 +1,251 @@
+        try:
+            page.wait_for_selector(verifier, timeout=timeout)
+        except playwright.sync_api.TimeoutError as e:
+            if timeout >= timeout_ms_max:
+                raise  # giving up
+            timeout *= 2
+        else:
+            break  # success
+
+
+def workload(user):
+    username = user["username"]
+    email = f"jhutar+{username}@redhat.com"
+
+    # Keep generating password until it contains both digit and letter.
+    # Given special char ("-") is granted, this will meet password requirements.
+    while True:
+        password = str(uuid.uuid4())
+        if any(char.isdigit() for char in password) and any(
+            char.isalpha() for char in password
+        ):
+            break
+
+    with playwright.sync_api.sync_playwright() as p:
+        browser = p.chromium.launch(
+            headless=PLAYWRIGHT_HEADLESS,
+        )
+        context = browser.new_context(
+            record_video_dir=PLAYWRIGHT_VIDEO_DIR,
+        )
+        page = context.new_page()
+
+        page.goto("https://console.dev.redhat.com")
+        page.wait_for_url("https://sso.redhat.com/**")
+
+        # Accept cookies
+        cookies_iframe = page.frame_locator('iframe[name="trustarc_cm"]')
+        cookies_button = cookies_iframe.get_by_role(
+            "button", name="Agree and proceed with standard settings"
+        )
+        cookies_button.click()
+
+        # Go to registration form
+        click_and_wait_hard(
+            page,
+            '//a[@id="rh-login-registration-link"]',
+            '//h1[text()="Register for a Red Hat account"]',
+        )
+
+        # Fill the form
+        user_input = page.locator('//input[@id="username"]')
+        user_input.fill(username)
+        pass_input = page.locator('//input[@id="password"]')
+        pass_input.fill(password)
+        first_input = page.locator('//input[@id="firstName"]')
+        first_input.fill("Performance")
+        last_input = page.locator('//input[@id="lastName"]')
+        last_input.fill("Testing")
+        email_input = page.locator('//input[@id="email"]')
+        email_input.fill(email)
+        terms_checkbox = page.locator(
+            '//input[contains(@id, "user.attributes.tcacc-SSO/ssoSignIn/")]'
+        )
+        terms_checkbox.click()
+        submit_button = page.locator('//input[@id="regform-submit"]')
+        submit_button.click()
+        page.wait_for_selector('//h1[text()="Email address verification"]')
+
+        # Verify account
+        verification_link = get_verification_link(email)
+        page.goto(verification_link)
+        page.wait_for_selector(
+            f'//h1[text()="Confirm validity of e-mail address {email}."]'
+        )
+        finish_button = page.locator('//a[text()="Finish"]')
+        finish_button.click()
+        page.wait_for_selector('//h1[text()="Your email address has been verified"]')
+
+        page.close()
+        browser.close()
+
+        return user
+
```

Comment:
workload function quite long one, can we consider breaking it 
for better follow up and readability  ?
maybe generating the passwords as a separate function

Reply:
Wanted to avoid it, but as you mentioned it :)

Reply:
Hope it is better, take a look now please.